centos-atomic-7.ks

```diff
@@ -0,0 +1,168 @@
+# This is a minimal CentOS install to serve as the docker atomic container image.
+#
+# To keep this image minimal it only installs English language. You need to change
+# dnf configuration in order to enable other languages.
+
+url --url="http://mirrors.kernel.org/centos/7/os/x86_64/"
+install
+bootloader --disabled
+timezone --isUtc --nontp UTC
+rootpw --lock --iscrypted locked
+
+keyboard us
+lang en_US
+
+firewall --disabled
+network --bootproto=dhcp --device=link --activate --onboot=on
+
+shutdown
+
+# Disk setup
+zerombr
+clearpart --all --initlabel
+part / --size 3000 --fstype ext4 --grow
+
+# Add nessasary repo for microdnf
+repo --name="microdnf" --baseurl="https://buildlogs.centos.org/cah-0.0.1" --cost=100
```

Comment:
we dont want to use a buildlogs repo in the official builds, is there another source for this content ?